tests/data/testloans.json

```diff
@@ -1,13 +1,92 @@
 [
```

Comment:
I would rename the file to just `loans.json`.

Reply:
ok